mkosi/__init__.py

```diff
@@ -3122,7 +3122,11 @@ def install_gentoo(
         # Please don't move, needs to be called before installing dracut
         # dracut is part of gentoo_pkgs_boot
         configure_dracut(args, packages={"dracut"}, root=root)
-        gentoo.invoke_emerge(args, root, pkgs=gentoo.pkgs_boot)
+        # Installing sys-kernel/installkernel-systemd-boot fails spectacularly
+        # due to beeing soft blocked by sys-kernel/installkernel-gentoo (which
+        # is pulled in by debianutils) unless --update and --deep is also
```

Comment:
From the PR description I understand that there's two competing `sys-kernel/install-foo`. I don't really speak Gentoo, so I'm not sure what "unmerge" means, but I guess something like "uninstall" or "removing from the transaction" (please correct me if I'm wrong). Great if `--update` and `--deep` fix this! Could you maybe make the comment slightly longer why that is or link to the appropriate manpage, so that us non-Gentoo people have an easier time to follow the reasoning?

Reply:
Will do.